end-to-end-tests/specs/oncoprinter.screenshot.spec.js

```diff
@@ -0,0 +1,169 @@
+
+        pasteToElement(".oncoprinterSamplesInput", sampleList);
```

Comment:
we used to use setText.  it took >30secs to set a large block text because it does character by character.  so we should use this method whenever we have to input substantial amounts of text to a field